edweiss-app/components/lectures/remotecontrol/modal.tsx

```diff
@@ -519,3 +520,116 @@ export const QuestionBroadcastModal: ReactComponent<{
         </ModalContainer>
     );
 };
+
+
+
+// ------------------------------------------------------------
+// ---------------      Quiz Broadcast Modal     --------------
+// ------------------------------------------------------------
+
+export const QuizBroadcastModal: ReactComponent<{
+    modalRef: React.RefObject<BottomSheetModalMethods>;
+    id: string;
+    quizModel?: LectureQuizzes.LectureQuiz;
+    courseId: string,
+    lectureId: string,
+    broadcasted: string,
+    setBroadcasted: React.Dispatch<React.SetStateAction<string>>;
+    onClose: () => void;
+}> = ({ modalRef, id, courseId, lectureId, quizModel, broadcasted, setBroadcasted, onClose }) => {
+
+
+    // Hooks
+    const [broadLoading, setBroadLoading] = useState(false);
+    const [showLoading, setShowLoading] = useState(false);
+    const [showResultToStudent, setShowResultToStudent] = useState(false);
+
+    useEffect(() => {
+        if (quizModel?.showResultToStudents !== undefined) {
```

Comment:
I do not think you need the !== undefined, just if (quizModel?.showResultToStudents) is enough

Reply:
Yep you are right my bad. I will fix it right away.

Reply:
All good